src/escape.js

```diff
@@ -0,0 +1,11 @@
+/* If the SVG has text that has curly braces, or
+if there is a <style> element (which will, of
+course, have curly braces) then we need to
+escape those for JSX parsing. */
+export default function escape(raw) {
```

Comment:
I looked for one.  There is not one that I can see.  If you can find one that does this simple one liner, I'm all open to that.